integration-test/pom.xml

```diff
@@ -124,84 +83,152 @@
+                        </goals>
+                    </execution>
+                </executions>
             </plugin>
             <plugin>
                 <groupId>org.apache.maven.plugins</groupId>
-                <artifactId>maven-jar-plugin</artifactId>
-                <version>3.2.0</version>
+                <artifactId>maven-antrun-plugin</artifactId>
                 <executions>
                     <execution>
-                        <id>default-jar</id>
-                        <phase/>
+                        <id>create-host-music-folder-for-docker</id>
+                        <phase>process-resources</phase>
                         <configuration>
-                            <finalName>unwanted</finalName>
-                            <classifier>unwanted</classifier>
+                            <tasks>
+                                <delete dir="${docker.testing.default-music-folder}"/>
+                                <mkdir dir="${docker.testing.default-music-folder}"/>
+                                <delete dir="${docker.testing.output-folder}"/>
+                                <mkdir dir="${docker.testing.output-folder}"/>
+                            </tasks>
                         </configuration>
+                        <goals>
+                            <goal>run</goal>
+                        </goals>
                     </execution>
                 </executions>
             </plugin>
             <plugin>
                 <groupId>org.apache.maven.plugins</groupId>
-                <artifactId>maven-surefire-plugin</artifactId>
-                <configuration>
-                    <forkCount>1</forkCount>
-                    <reuseForks>false</reuseForks>
-                    <includes>
-                        <include>**/Parallel*IT.class</include>
-                    </includes>
-                </configuration>
+                <artifactId>maven-failsafe-plugin</artifactId>
+                <version>2.22.2</version>
+                <executions>
+                    <execution>
+                        <configuration>
+                            <!-- Skip integration tests by adding -Dfailsafe.skip to the Maven command-line -->
+                            <skip>${failsafe.skip}</skip>
+                            <systemPropertyVariables>
+                                <DockerTestingContainer>${docker.testing.container}</DockerTestingContainer>
+                                <DockerTestingHost>${docker.testing.host}</DockerTestingHost>
+                                <!-- This is the folder on the host (not on the container), used in org.airsonic.test.Scanner -->
+                                <DockerTestingDefaultMusicFolder>${docker.testing.default-music-folder}</DockerTestingDefaultMusicFolder>
+                                <IntegrationTestsOutputDirectory>${project.build.directory}/data/responses</IntegrationTestsOutputDirectory>
```

Comment:
Not sure this guy is needed? You can just retrieve responses and compare them in-memory

Reply:
See my other comment.